messages/config-command.md

```diff
@@ -0,0 +1,74 @@
+# command.summary
+
+Display the current state of configuration for Code Analyzer.
+
+# command.description
+
+Code Analyzer gives you the ability to configure settings that modify Code Analyzer's behavior, to override the tags and severity levels of rules, and to configure the engine specific settings.  Use this command to see the current state of this configuration and to save this state to a YAML-formatted file that you can modify to our needs.
+
+To apply a custom configuration with Code Analyzer, either keep your custom configuration settings in a 'code-analyzer.yml' file located in the current working directory from which you are executing commands, or specify the location of your custom configuration file to each of the Code Analyzer commands via the `--config-file` flag.
+
+# command.examples
+
+- Display the current state configuration for Code Analyzer based on the default recommendations, and a local `code-analyzer.yml` file if such a file exists: 
+
+     <%= config.bin %> <%= command.id %>
+
+- This example is identical to the previous one, if `./code-analyzer.yml` is a real file.
+
+     <%= config.bin %> <%= command.id %> --config-file ./code-analyzer.yml --rule-selector Recommended
+
+- Write the current state of configuration to `code-analyzer.yml`, including any configuration from an existing `code-analyzer.yml` file. This preserves all values from the original config, but will overwrite any comments:
+
+     <%= config.bin %> <%= command.id %> --config-file ./code-analyzer.yml --output-file code-analyzer.yml
+
+- This example is equivalent to the previous one:
+
+     <%= config.bin %> <%= command.id %> --output-file code-analyzer.yml
+
+- Display the configuration state for all rules, instead of just the recommended ones:
+
+     <%= config.bin %> <%= command.id %> --rule-selector all
+- Display the configuration state associated with recommended rules that are applicable to your workspace folder, `./src`:
+
+     <%= config.bin %> <%= command.id %> --workspace ./src
+
```

Comment:
Any reason why you didn't include my last 2 examples? I don't see any example showing how you can do an intersection `':'` on the rule selection? And my last example demonstrates a use case that takes all flags into account in one go... which I believe is a good concluding example to put all the concepts together. Anyway, that's just my 2 cents I guess.